### ISSUE TITLE
Computed Field Comments Do Not Propagate to GraphQL Schema Descriptions

### ISSUE THREAD

**Issue by lummish** [hasura/graphql-engine#6256]
## Descriptions
Currently, it appears as though comments applied to computed fields via the console do not propagate to GraphQL schema descriptions. Instead, a standard description is applied, as seen below:

```gql
type table {
  # A computed field, executes function "with_args_function"
  #
  # Arguments
  # args: input parameters for function "with_args_function"
  with_args_computed_field(args: with_args_computed_field_args!): Boolean

  # A computed field, executes function "no_args_function"
  no_args_computed_field: Boolean
} 
```

Ideally, first line of the description would instead be the user-defined comment.

## Related Issues
#446 #2397

**Comment by bitjson** [comment]
Additionally, it would be great if the `COMMENT` on the underlying SQL function could be used by Hasura in the same way table and column `COMMENT`s are propagated. E.g.
```sql
CREATE FUNCTION transaction_data_carrier_outputs(transaction_row transaction) RETURNS SETOF output
  LANGUAGE sql STABLE
AS $$
  SELECT * FROM output WHERE transaction_hash = $1.hash AND (value_satoshis = 0 OR get_byte(locking_bytecode, 0) = 106);
$$;
COMMENT ON FUNCTION transaction_data_carrier_outputs (transaction) IS 'Return all of this transaction''s "data carrier" outputs: outputs in which value_satoshis is 0 or locking_bytecode begins with OP_RETURN.';
```

It would be great if Hasura used this underlying comment for the computed field if no other Hasura `Comment:` is specified.

**Comment by bitjson** [comment]
I'm not sure which release included this, but it seems to have been solved as of `v2.0.9`. 👍

`COMMENT`s on underlying SQL functions also propagate to the GraphQL Schema 🚀 (e.g. the [`data_carrier_outputs` field from above](https://github.com/bitauth/chaingraph/blob/cbebedefea908957b0373d77a60ec17fdff2050b/docs/generated-schema.md#transaction))

**Comment by daniel-chambers** [comment]
@lummish In `v2.3.0-beta.1` we've added functionality so that the comment you set in the console on computed fields is used as the GraphQL schema description for that field. If you don't set a comment, it will use the comment set on the underlying SQL function (as @bitjson spoke about above), and if _that_ does not exist, it will use an autogenerated description as described in the OP.

* API doco: https://hasura.io/docs/latest/graphql/core/api-reference/metadata-api/computed-field.html#args-syntax (see the `comment` arg).
* Commit: https://github.com/hasura/graphql-engine/commit/2c7a4e3a1645fe38959c4ab83aad80ab17dacb9a

Enjoy! 😀
